### DISCUSSION TITLE
How to set up JMAP authentication?

### DISCUSSION THREAD

**Discussion by jet10000** [stalwartlabs/jmap-server#62] [General]
<img width="490" alt="image" src="https://github.com/stalwartlabs/jmap-server/assets/2258120/dcf33367-d0ce-45e5-a25c-872c728d0b8b">

**Comment by mdecimus** [comment]
You can read about directory configuration here https://stalw.art/docs/directory/overview

**Reply by jet10000** [reply]
I have configured basic auth with account and password. How do I configure `bearer` authentication?

**Reply by mdecimus** [reply]
There is no need to configure `Bearer` authentication on Stalwart. You just need to tell your HTTP client to authenticate the HTTP requests.
